package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.1",
   "description": "Cracking the Coding Interview - Answers in Javascript",
   "scripts": {},
```

Comment:
Notice that a bunch of errors show up when I run eslint. Will fix them over the course of this week.